src/App.tsx

```diff
@@ -10,21 +10,32 @@ import { Outlet } from "react-router-dom"
 
 import getClient from "./Client"
 import * as consts from "./Consts"
+import getUser from "./User"
```

Comment:
I think at this stage, we don't require any changes at all to `App.tsx`. We have a global utility for learning user ID, but nothing in this file needs to know the user ID.

Reply:
So, Are you saying that we should remove the loggedIn state from this component ? and every component should directly access the global function ? 

and just to specify, this means that we should also stop passing loggedIn state as props to LoginControl ? If what I am saying is correct and is the expected result, then I will make the change asap. 

Then it would be nice if you can clarify what changes should I make. Thanks :)

Reply:
Nope. Just saying that I think `App.tsx` is fine to restore to its original state -- where logged in / out is a boolean and it's passed to the login control.